rad/src/operators/integer.rs

```diff
@@ -0,0 +1,245 @@
+use serde_cbor::value::{from_value, Value};
+use std::{borrow::ToOwned, convert::TryFrom};
+
+use crate::{
+    error::RadError,
+    types::{
+        boolean::RadonBoolean, bytes::RadonBytes, float::RadonFloat, integer::RadonInteger,
+        string::RadonString, RadonType,
+    },
+};
+
+pub fn absolute(input: &RadonInteger) -> Result<RadonInteger, RadError> {
+    let result = input.value().checked_abs();
+
+    if let Some(result) = result {
+        Ok(RadonInteger::from(result))
+    } else {
+        Err(RadError::Overflow)
+    }
+}
+
+pub fn to_float(input: RadonInteger) -> Result<RadonFloat, RadError> {
+    RadonFloat::try_from(Value::Integer(input.value()))
+}
+
+pub fn to_bytes(input: RadonInteger) -> RadonBytes {
+    RadonBytes::from(Value::Integer(input.value()))
+}
+
+pub fn to_string(input: RadonInteger) -> Result<RadonString, RadError> {
+    RadonString::try_from(Value::Text(input.value().to_string()))
+}
+
+pub fn multiply(input: &RadonInteger, args: &[Value]) -> Result<RadonInteger, RadError> {
+    let wrong_args = || RadError::WrongArguments {
+        input_type: RadonInteger::radon_type_name(),
+        operator: "Multiply".to_string(),
+        args: args.to_vec(),
+    };
+
+    let arg = args.first().ok_or_else(wrong_args)?.to_owned();
+    let multiplier = from_value::<i128>(arg).map_err(|_| wrong_args())?;
+    let result = input.value().checked_mul(multiplier);
+
+    if let Some(result) = result {
+        Ok(RadonInteger::from(result))
+    } else {
+        Err(RadError::Overflow)
+    }
+}
+
+pub fn greater_than(input: &RadonInteger, args: &[Value]) -> Result<RadonBoolean, RadError> {
+    let wrong_args = || RadError::WrongArguments {
+        input_type: RadonInteger::radon_type_name(),
+        operator: "GreaterThan".to_string(),
+        args: args.to_vec(),
+    };
+
+    let arg = args.first().ok_or_else(wrong_args)?.to_owned();
+    let other = from_value::<i128>(arg).map_err(|_| wrong_args())?;
+    Ok(RadonBoolean::from(input.value() > other))
+}
+
+pub fn less_than(input: &RadonInteger, args: &[Value]) -> Result<RadonBoolean, RadError> {
+    let wrong_args = || RadError::WrongArguments {
+        input_type: RadonInteger::radon_type_name(),
+        operator: "LessThan".to_string(),
+        args: args.to_vec(),
+    };
+
+    let arg = args.first().ok_or_else(wrong_args)?.to_owned();
+    let other = from_value::<i128>(arg).map_err(|_| wrong_args())?;
+    Ok(RadonBoolean::from(input.value() < other))
+}
+
+pub fn modulo(input: &RadonInteger, args: &[Value]) -> Result<RadonInteger, RadError> {
+    let wrong_args = || RadError::WrongArguments {
+        input_type: RadonInteger::radon_type_name(),
+        operator: "Modulo".to_string(),
+        args: args.to_vec(),
+    };
+
+    let arg = args.first().ok_or_else(wrong_args)?.to_owned();
+    let modulo = from_value::<i128>(arg).map_err(|_| wrong_args())?;
+    Ok(RadonInteger::from(input.value() % modulo))
```

Comment:
Actually, in Rust `%` is the remainder operator, not the modulo operator. The main difference is what happens with negative numbers: `-1 mod 2 = 1` but `-1 rem 2 = -1`. I assume we want to keep the standard behavior, so there is a method which can be used as the modulo operator: `rem_euclid`.

Additionally, performing a modulo with 0 will panic at runtime, so we should use the checked variant of the method:
```suggestion
    Ok(RadonInteger::from(input.value().checked_rem_euclid(modulo)))
```